src/main/resources/schema.graphqls

```diff
@@ -1978,7 +1975,9 @@ type Mutation {
     """Update the omero project of a work."""
     updateWorkOmeroProject(workNumber: String!, omeroProject: String): Work!
     """Update the DNAP study of a work."""
-    updateWorkDnapProject(workNumber: String!, dnapStudy: String): Work!
+    updateWorkDnapProject(workNumber: String!, ssStudyId: Int): Work!
+    """Updates Stan's internal list of Dnap Studies, and returns the enabled ones."""
+    updateDnapStudies: [DnapStudy!]!
```

Comment:
I hope this mutation is to refresh/update Stan's studies from the latest mlwh ones.

Reply:
That's right. They will be updated automatically, but if someone (admin) needs to update them immediately, this provides the option.

Reply:
CoolFor client side- Do you think, this need to be called whenever

Reply:
It would make sense to have a button somewhere that an admin could press to force the update

Reply:
It would be expensive to do it each time. We really only want to do an unscheduled update if someone requests it.

Reply:
Ok 👍

Reply:
It causes Stan to read the mlwh and update its internal list of studies. Since it involves updating information (and because it's triggering an expensive task), it's a mutation.